get-started/code-experiments/src/util.py

```diff
@@ -0,0 +1,92 @@
+from ruamel.yaml import YAML
+import numpy as np
+
+FULL_PARAMS = {
```

Comment:
simplicity of the code is important to my mind ... do we actually okay to keep this workaround?